Update members for W20

kylemtam opened the pull request:
- Update titles from requested members
- Add new members according to the spreadsheet
- Modify Team page to mention members from engineering, science and math faculties

kylemtam committed:
Added new members for W2020

- Added ABM, Hamza, Jared, Joanna, Kaelan, Kaveeshan, Lana, Nadine, Sophie
R, Waleed, Zhen Bo, Ben, Ronald, Sophie H, Francis, Chamath, Tommy and
Justin (alumni)
- Purged Vincent, Em and Colin
- Moved Nick to alumni :(
- Moved Tom and Shirley up as team leads
- Adjusted photos so maximum file size is under 700KB to quicken photo
loading
- Added "science and mathematics faculties" in the first paragraph
- Adjusted titles of existing members to reflect their current roles

kylemtam committed:
Added Sophie H.

Fixed capitalization issue in first paragraph

kylemtam committed:
Added photos

Included updated photos for Doris, Justin, Jared
Added photos for Ben, Chamath, Francis, Nadine, Ronald and Tommy

jacobdeery commented:
For people with particularly long names, the title gets uncomfortably close to the bottom of the image. Is there a way that this could be adjusted in the CSS?

![image](https://user-images.githubusercontent.com/13474696/74397145-04669200-4de2-11ea-9f33-561c20ae99c5.png)

AidanHa reviewed (approved):
LGTM,  don't see anything else standing out

Commit:
Fix CSS issue where long names overflow profile

Previously, the position of member and alumni titles had a minimum
distance from the top, causing long names to overflow out of the
bottom of the profile image.
Changed positioning to be relative to the bottom of the parent element
to force text to overflow upwards.
Also reorganized unnecessary title and year divs into header tags
instead, since the only real difference between them was font
sizing.

kylemtam committed:
Add Wendi to Team page

jacobdeery reviewed (approved):
LGTM pending Kyle's requested change

Commit:
Fix mismatched h2/h1 header tags
